src/Parser/MarksmanshipHunter/Modules/FocusChart/PassiveFocusWasted.js

```diff
@@ -0,0 +1,25 @@
+import Module from 'Parser/Core/Module';
+
+class PassiveFocusWasted extends Module {
+
+	cappedTimer = [];
+	tracker = 0; //to tell if cappedTimer prop has updated, since we can't compare arrays
+
+  on_byPlayer_cast(event){
+		if (event.sourceID === this.owner.player.id &&(event.type === 'cast' || event.type === 'energize') ){
+			this.tracker++;
+			if(!this.cappedTimer){
+			}
+			const secIntoFight = (event.timestamp - this.owner.fight.start_time);
+			if(!this.cappedTimer[secIntoFight]){
+				this.cappedTimer[secIntoFight] = event.classResources[0]['amount'];
```

Comment:
see my previous classResource comment.

iirc the `amount` is the focus before the spell's cost from the cast event was applied, so you generally want to use `classResource.amount - classResource.cost`.